cmd/rigs/cmd/publish_renders_index.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	rendersIndexCmd.Flags().String("out", "./index.car", "filename to write the index CAR to")
+
+	publishCmd.AddCommand(rendersIndexCmd)
+}
+
+var rendersIndexCmd = &cobra.Command{
```

Comment:
New command that creates a CAR index of all the rig ids and their corresponding imagery cids. It's basically a map of rig id to cid. The root cid of the map can then be pinned or unpinned on any pinning service and all the cids in the map will also be pinned/unpinned. This is much easier than managing 3000 individual cids.